pkg/network/ebpf/c/protocols/protocol-dispatcher-helpers.h

```diff
@@ -42,7 +46,7 @@ static __always_inline void protocol_dispatcher_entrypoint(struct __sk_buff *skb
     }
 
     // We don't process non tcp packets, nor empty tcp packets which are not tcp termination packets.
```

Comment:
update the comment here, mention that we filter packets that have ONLY ack flag est

Reply:
done